documentation/src/main/asciidoc/topics/con_write_behind.adoc

```diff
@@ -1,15 +1,14 @@
 [id='write_behind']
 = Write-Behind
-Write-Behind is an cache writing mode where writes to memory are synchronous
-and writes to cache stores are asynchronous. When a client application
-updates a cache entry, {brandname} adds the update to a modification queue and
-then modifies the cache store in a different thread than the client thread.
-
-You can configure the number of threads that consume the modification queue and
-apply updates to the underlying cache store. The modification queue fills up if
-there are not enough threads to handle the updates or if the underlying cache
-store becomes unavailable. When this occurs, {brandname} uses Write-Through
-mode until the modification queue can accept new entries.
+Write-Behind is a cache writing mode where writes to memory are synchronous
+and writes to cache stores are asynchronous. When client applications
+update entries, {brandname} modifies cache stores in different threads to the
```

Comment:
We don't use different threads. That is up to the store implementation if it needs to use another thread.